README.md

```diff
@@ -368,6 +368,23 @@ Please see the 'Commit comment' section for more details.
 If it is set to `true`, this action automatically pushes the generated commit to GitHub Pages branch.
 Otherwise, you need to push it by your own. Please read 'Commit comment' section above for more details.
 
+#### `comment-always` (Optional)
+
+- Type: Boolean
+- Default: `false`
+
+If it is set to `true`, this action will leave a commit comment comparing the current benchmark with previous.
+`github-token` is necessary as well. Please note that a personal access token is not necessary to
+send a commit comment. `secrets.GITHUB_TOKEN` is sufficient.
+
+#### `save-on-pr` (Optional)
+
+- Type: Boolean
+- Default: `true`
+
+If it is set to `true`, any commits that are part of a PR will not save the current benchmark.
+You can use this option to set up your action to compare the benchmarks between PR and base branch.
```

Comment:
This is mostly about external data json. `auto-push` is only used for storing in pages. I need to compare PR's benchmark data with master's benchmark data. Currently, the logic is too much interwined with saving benchmarks because that's how we need to get previous data. I will look into this. And I will unify the option with `auto-push` and name it as `save` while maintaining backwards compatibility. What do you think?

Reply:
The logic is too much interwined. So, I just used the `auto-push` setting for saving to external data file also

Reply:
OK, would you make the `save-on-pr` not only for PR? I feel the feature is too specific if it is only for PR. Instead, would you make a `save-data-file` boolean option which controls whether saving the results to file or not. It is more general because some people may want to be notified by comment every commit on `push` event. In your use case, `if` is available to control the option in step as above example.